exec/java-exec/src/main/java/org/apache/drill/exec/util/VectorUtil.java

```diff
@@ -227,4 +233,69 @@ private static int getColumnWidth(int[] columnWidths, int columnIndex) {
     return (columnWidths == null) ? DEFAULT_COLUMN_WIDTH
         : (columnWidths.length > columnIndex) ? columnWidths[columnIndex] : columnWidths[0];
   }
+
+  /**
+   * Formats ValueVector elements in accordance with the corresponding system/session options.
+   *
+   * @param value ValueVector element to format, not null
+   * @param minorType the minor type of the element, not null
+   * @param options the OptionManager instance, not null
+   * @return the formatted value, null if failed
+   */
+  public static String formatValueVectorElement(Object value, TypeProtos.MinorType minorType, OptionManager options) {
+    String formattedValue = null;
+    switch (minorType) {
+      case TIMESTAMP:
+        if (value instanceof LocalDateTime) {
+          String formatPattern = options.getString(ExecConstants.WEB_TIMESTAMP_DISPLAY_FORMAT);
+          if (!formatPattern.isEmpty()) {
```

Comment:
Emptiness check can be done in `getDateTimeFormatter` value to avoid repetition.

Reply:
Moved this check to a separate method.